lib/code-renderer.js

```diff
@@ -0,0 +1,20 @@
+var React = require('react');
+var $ = React.createElement;
+
+var highlight = require('./highlight');
+
+module.exports = function codeRenderer(node, props, children) {
+  var lang = (node.info || '').trim() || null;
+  var className = 'language-' + lang;
+
+  var highlightedCode = highlight(lang, children[0]);
+
+  var codeElement = $('code', {
+    className: className,
+    dangerouslySetInnerHTML: {
```

Comment:
I'm not sure server rendering is a core use-case for devcards.

Even with server rendering, you need JS enabled to use devcards properly - and the `mount` lifecycle would still fire and enable the highlighting.

I might have a go this evening at what it'd look like with lifecycle hooks - then we can compare